app/models/post.rb

```diff
@@ -6,6 +6,7 @@ class Post < ActiveRecord::Base
   validates :text, length: { minimum: 3, maximum: 255 }
   validates :comments_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
   validates :likes_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
+  after_save :update_post_counter
 
```

Comment:
![image](https://user-images.githubusercontent.com/65068771/178789315-b936e9b2-6df9-4f6a-804c-1585fffd5f75.png)

- [ ]  Kindly note that this error occurred because of your association in your Post model you are referencing the user as `author_id` but in your database the post is referenced by the `user_id`. Kindly work on your association and fix this.
Your Post table 
```sql

  create_table "posts", force: :cascade do |t|
    t.string "title"
    t.text "text"
    t.integer "comments_counter", default: 0
    t.integer "likes_counter", default: 0
    t.datetime "created_at", null: false
    t.datetime "updated_at", null: false
    t.bigint "user_id", null: false
    t.index ["user_id"], name: "index_posts_on_user_id"
  end
```

Reply:
Hello again;

> I **really appreciate your feedback and have made changes accordingly. In this case, however, my schema doesn't look like this because my code reviewer asked me to change `user_id` to `author_id` at the first milestone, I updated it accordingly and added the "author_id" reference in all of its dependencies, so check the `schema` and `models`. It works fine in my local environment. Also, create the database according to the `schema`. Thank You again.**

* Here is the post table looks like
***
![image](https://user-images.githubusercontent.com/51437483/178795644-20363689-6fa3-4734-bf00-f4d9dc1205d7.png)

Reply:
Addressed